packages/vega-expression/index.js

```diff
@@ -14,7 +14,13 @@ export {
   default as ASTNode
 } from './src/ast';
 
-export { default as parse} from './src/parser';
-export { default as codegen } from './src/codegen';
+export {
+  default as parse,
+  default as parseExpression
```

Comment:
Add comment documenting why we need to export default twice?

Reply:
I think it's because Vega exports it this way as well. It's so that we can import from Vega or this specific package without changing the import.

Reply:
`default as parse` because that's what it used to be so we don't want to break backward compatibility

`default as parseExpression` because the vega package uses this name: https://github.com/vega/vega/blob/1f14aa2acfa34825e3809a7076553504b0ef90db/packages/vega/index.js#L93. In order to swap out a specific package for the vega bundle, we need to use the same names.

I added a comment.